tests/utils.py

```diff
@@ -36,8 +37,8 @@ def internal_sideeffect(*args, **kwargs):
         value = json.loads(value_str)
         del value["one_time_code"]
         python_args["body"] = json.dumps(value, sort_keys=True)
-        newargs = json.dumps(python_args, sort_keys=True)
-        arguments = (newargs,)
+        # newargs = json.dumps(python_args, sort_keys=True)
```

Comment:
let's snip these two lines out entirely (same comment in the try/except block directly below) since we have that logic after you create the `OrderedDict`

Reply:
Oops, will do so.